frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -524,15 +524,14 @@ const Profile: NextPage = () => {
       {/* confetti animation should be shown if user has sent first forwarded email, is a free user, and has not reached the max onboarding step */}
       {isFlagActive(runtimeData.data, "free_user_onboarding") &&
         !profile.has_premium &&
-        profile.forwarded_first_reply &&
-        profile.onboarding_state < freeOnboardingCelebrationStep && (
+        profile.onboarding_free_state < freeOnboardingCelebrationStep && (
```

Comment:
(Blocking) With the changes in #4171, this triggers the confetti animation once for existing free users who sign in normally (did not come from onboarding).